package.json

```diff
@@ -26,18 +26,17 @@
     "clean:docs": "shx rm -rf documentation",
     "clean:coverage": "shx rm -rf coverage",
     "doc": "npm run doc:generate && opn documentation/index.html",
-    "doc:generate": "npm run clean:docs && jsdoc -c .jsdocrc",
-    "eslint": "eslint -c .eslintrc.js src",
-    "eslint-quiet": "eslint -c .eslintrc.js --quiet src",
-    "eslint-fix": "eslint -c .eslintrc.js --fix src",
+    "doc:generate": "npm run clean:docs && typedoc",
+    "eslint": "eslint -c .eslintrc.js src --ext .ts",
+    "eslint-quiet": "eslint -c .eslintrc.js --quiet src --ext .ts",
```

Comment:
Can we set the extension in eslint config instead of in the script?

Reply:
No. https://eslint.org/docs/user-guide/configuring#specifying-processor 

> ESLint checks the file extension of named code blocks then ignores those if --ext CLI option didn’t include the file extension. Be sure to specify the --ext option if you wanted to lint named code blocks other than *.js.